src/access/tests/TestAccessControl.cpp

```diff
@@ -327,321 +370,285 @@ struct CheckData
     bool allow;
 };
 
-constexpr CheckData checkData1[] =
-{
+constexpr CheckData checkData1[] = {
     // Checks for entry 0
-    {
-        .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kCase, .subjects = { 0x1111111111111111 }, },
-        .requestPath = { .cluster = kAccessControlCluster, .endpoint = 0 },
-        .privilege = Privilege::kAdminister,
-        .allow = true
-    },
-    {
-        .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kCase, .subjects = { 0x1111111111111111 }, },
-        .requestPath = { .cluster = 1, .endpoint = 2 },
-        .privilege = Privilege::kManage,
-        .allow = true
-    },
-    {
-        .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kCase, .subjects = { 0x1111111111111111 }, },
-        .requestPath = { .cluster = 3, .endpoint = 4 },
-        .privilege = Privilege::kOperate,
-        .allow = true
-    },
-    {
-        .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kCase, .subjects = { 0x1111111111111111 }, },
-        .requestPath = { .cluster = 5, .endpoint = 6 },
-        .privilege = Privilege::kView,
-        .allow = true
-    },
-    {
-        .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kCase, .subjects = { 0x1111111111111111 }, },
-        .requestPath = { .cluster = 7, .endpoint = 8 },
-        .privilege = Privilege::kProxyView,
-        .allow = true
-    },
-    {
-        .subjectDescriptor = { .fabricIndex = 2, .authMode = AuthMode::kCase, .subjects = { 0x1111111111111111 }, },
-        .requestPath = { .cluster = 1, .endpoint = 2 },
-        .privilege = Privilege::kAdminister,
-        .allow = false
-    },
-    {
-        .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kPase, .subjects = { 0x1111111111111111 }, },
-        .requestPath = { .cluster = 1, .endpoint = 2 },
-        .privilege = Privilege::kAdminister,
-        .allow = false
-    },
-    {
-        .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kGroup, .subjects = { 0x1111111111111111 }, },
-        .requestPath = { .cluster = 1, .endpoint = 2 },
-        .privilege = Privilege::kAdminister,
-        .allow = false
-    },
-    {
-        .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kCase, .subjects = { 0x2222222222222222 }, },
-        .requestPath = { .cluster = 1, .endpoint = 2 },
-        .privilege = Privilege::kAdminister,
-        .allow = false
-    },
+    { .subjectDescriptor = { .fabricIndex = 1, .authMode = AuthMode::kCase, .subject = kOperationalNodeId3 },
```

Comment:
Something up with the restyling? (End brace seems to have moved.)

Reply:
Yes, restyling changed the structure. I didn't change anything in the existing testcases. Just added new testcases for CATs.